### PR TITLE
Changing the UI and backend according to the wireframe [https://githu…

### PR DESCRIPTION
**Pull request by DMHP**
…b.com/wso2-dev-ux/product-is/blob/master/Wireframes/admin-portal/v4/3.30_Admin_forced_Reset_password.png]

### CODE REVIEW

File: portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/pages/reset/admin-initiated-password-reset.hbs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+{{#fillZone "content"}}
+    <div class="body-wrapper">
+        <div class="page-header">
+            <h1>{{i18n "recovery.reset.password"}}</h1>
+        </div>
+        <form class="form-horizontal" role="form" method="post">
+
+            <label class="control-label">Select password reset method</label>
```
Comment (GayanM) [inline portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/pages/reset/admin-initiated-password-reset.hbs]: apply internalization

File: portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/pages/reset/admin-initiated-password-reset.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            ["user1"]);
+    } catch (e) {
+        return {errorMessage: 'Error while generating the passcode'};
+    }
+    return passcode;
+}
+
+function onPost(env){
+    var passcode = getPasscode(env);
+    sendToClient("passCode", passcode);
```
Comment (nuwandiw) [inline portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/pages/reset/admin-initiated-password-reset.js]: getPasscode(env) might return an errorMessage. Need to handle if (passcode.errorMessage) case and give front end alert if it is true.

File: portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/RecoveryMgtServiceImpl.java [hunk 3]
```diff
@@ -273,16 +273,17 @@ public boolean verifyUsername(Map<String, String> userClaims) throws IdentityRec
...
-     * @param passCode generated one time password
      * @throws UserPortalUIException
      */
     @Override
-    public void persistPassCode(String uniqueUserId, String passCode) throws UserPortalUIException {
+    public String persistPasscode(String uniqueUserId) throws UserPortalUIException {
+        String passcode = null;
         try {
-            AdminForcePasswordResetManager.getInstance().persistPasscode(uniqueUserId, passCode);
+            passcode = AdminForcePasswordResetManager.getInstance().persistPasscode(uniqueUserId);
```
Comment (GayanM) [inline portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/RecoveryMgtServiceImpl.java]: Directly call util method to generate password

**Comment by sagara-gunathunga** [comment]
@johannnallathamby Please take a look

**Comment by DMHP** [comment]
https://github.com/wso2/product-is/pull/1214

**Pull request closed without merging**